specgen_connector_migrator/internal/workflow_release.go

```diff
@@ -0,0 +1,71 @@
+// Copyright © 2025 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal
+
+import (
+	"embed"
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+//go:embed release.yaml
+var workflowFiles embed.FS
+
+type WorkflowRelease struct{}
+
+func (w WorkflowRelease) Migrate(workingDir string) error {
+	// Check for both possible file extensions
+	possiblePaths := []string{
+		filepath.Join(workingDir, ".github", "workflows", "release.yml"),
+		filepath.Join(workingDir, ".github", "workflows", "release.yaml"),
+	}
+
+	// Check if either file exists
+	var existingPath string
+	for _, path := range possiblePaths {
+		if _, err := os.Stat(path); err == nil {
+			existingPath = path
+			break
+		}
+	}
```

Comment:
Why don't we just overwrite the existing file if it exists, instead of deleting the old one and creating a new one?
```suggestion
	targetPath := possiblePaths[0]
	for _, path := range possiblePaths {
		if _, err := os.Stat(path); err == nil {
			targetPath = path
			break
		}
	}
```

Reply:
I think I wanted all files to be named in the same way, for making future work easier potentially. But overwriting works too.:)